crates/cast/src/cmd/logs.rs

```diff
@@ -77,7 +77,7 @@ impl LogsArgs {
         if !subscribe {
             let logs = cast.filter_logs(filter).await?;
             sh_println!("{logs}")?;
-            return Ok(())
+            return Ok(());
```

Comment:
Please undo the semicolon insertions, see the contribution guide for clippy / fmt, we use a nightly version for them